framework/src/problems/FEProblemBase.C

```diff
@@ -2141,7 +2141,7 @@ FEProblemBase::reinitElemNeighborAndLowerD(const Elem * elem, unsigned int side,
       std::vector<Point> reference_points;
       FEInterface::inverse_map(
           lower_d_elem_neighbor->dim(), FEType(), lower_d_elem_neighbor, qps, reference_points);
-      reinitLowerDElem(lower_d_elem_neighbor, tid, &qps);
+      reinitLowerDElem(lower_d_elem_neighbor, tid, &reference_points);
```

Comment:
Whoa.  How did nobody catch this bug before?

Reply:
I don't know. I suspect the coverage report is going to tell me this line isn't hit. I suspect that we currently only ever call this method with the element that has the lower-D element associated with it, e.g. for non-refined meshes we are calling with the lower ID higher dimensional element etc.